### PR TITLE
Fix Hotkey Toggle button

### PR DESCRIPTION
**Pull request by Markolie**
**What does this PR do:**
Fixes the toggle hotkey button not working properly. After making the AZERTY change I hadn't tested this properly.

**Changelog:**
:cl: Markolie
fix: An issue where the Hotkey Toggle button wasn't working has been fixed.
/:cl:

### CODE REVIEW

File: code/datums/helper_datums/hotkey_modes.dm [hunk 1]
```diff
@@ -2,6 +2,14 @@
...
 	var/macro_hotkeys_inactive
 	var/macro_hotkeys_active
+	var/mob/my_mob
+
+/datum/hotkey_mode/New(mob)
+	my_mob = mob
+
+/datum/hotkey_mode/proc/set_winset_values()
+	winset(my_mob, null, "mainwindow.macro=[macro_hotkeys_inactive] hotkey_toggle.is-checked=false input.focus=true input.background-color=#d3b5b5")
+	winset(my_mob, null, "hotkey_toggle.command=\".winset \\\"mainwindow.macro != [macro_hotkeys_inactive] ? mainwindow.macro=[macro_hotkeys_inactive] hotkey_toggle.is-checked=false input.focus=true input.background-color=#d3b5b5 : mainwindow.macro=[macro_hotkeys_active] hotkey_toggle.is-checked=true mapwindow.map.focus=true input.background-color=#f0f0f0\\\"\"")
```
Comment (Markolie) [inline code/datums/helper_datums/hotkey_modes.dm]: If you won't ask, I won't tell. These escaped quotes and slashes is the only way to get this to work.

File: interface/skin.dmf [hunk 2]
```diff
@@ -1543,7 +1541,7 @@ window "mainwindow"
 		saved-params = ""
 		is-flat = false
 		text = "Hotkey Toggle"
-		command = ".winset \"mainwindow.macro != mainwindow.macro_hotkey_mode_inactive ? mainwindow.macro=mainwindow.macro_hotkey_mode_inactive hotkey_toggle.is-checked=false input.focus=true input.background-color=#d3b5b5 : mainwindow.macro=mainwindow.macro_hotkey_mode_active hotkey_toggle.is-checked=true mapwindow.map.focus=true input.background-color=#f0f0f0\""
+		command = ".winset \"mainwindow.macro != macro ? mainwindow.macro=macro hotkey_toggle.is-checked=false input.focus=true input.background-color=#d3b5b5 : mainwindow.macro=hotkeymode hotkey_toggle.is-checked=true mapwindow.map.focus=true input.background-color=#f0f0f0\""
```
Comment (Markolie) [inline interface/skin.dmf]: So, in BYOND you can actually SET window values by just calling <windowname>.<variable> but you can't actually get them using the same method.

File: code/datums/helper_datums/hotkey_modes.dm (hunk 1 above)
Comment (Crazylemon64) [reply code/datums/helper_datums/hotkey_modes.dm]: It makes sense, you'd need to do the same for declaring attributes of an HTML tag, for example.

**Comment by Fox-McCloud** [comment]
This is still broken.

I cannot control click to start pulling things, I can't click drag to get onto a table, and my movement will just flat out break, at times.

**Pull request merged**